### PR TITLE
Switch to Node.js protobufjs-cli

### PR DESCRIPTION
**Pull request by diceride**
* Use the Node.js the [protobufjs-cli@1.0.0](https://www.npmjs.com/package/protobufjs-cli) command line interface for static code and TypeScript definition generation.

    https://github.com/protobufjs/protobuf.js/releases/tag/protobufjs-cli-v1.0.0

* Update the Node.js dependency [protobufjs@6.10.3](https://www.npmjs.com/package/protobufjs) -> [protobufjs@7.0.0](https://www.npmjs.com/package/protobufjs).

    https://github.com/protobufjs/protobuf.js/releases/tag/protobufjs-v7.0.0

* Chore: Remove obsolete protobuf.js dependencies.

### CODE REVIEW

**Comment by bduffany** [comment]
Thank you for this contribution!

Looks like the build is failing, and I also noticed a runtime error once the build is fixed. Could you apply this patch? https://github.com/diceride/buildbuddy/compare/protobufjs-cli...buildbuddy-io:buildbuddy:protobufjs-cli

**Comment by diceride** [comment]
It seems that some tests are flaky.

File: proto/defs.bzl [hunk 1]
```diff
@@ -2,31 +2,9 @@ load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
 
 # TODO switch to protobufjs-cli when its published
 # https://github.com/protobufjs/protobuf.js/commit/da34f43ccd51ad97017e139f137521782f5ef119
```
Comment (bduffany) [inline proto/defs.bzl:4]:
```suggestion
```

**Review by bduffany — APPROVED** [review]
Thanks again!

**Pull request merged**